src/storage/sftp/storage.c

```diff
@@ -867,8 +889,8 @@ storageSftpNew(const String *const path, const String *const host, const unsigne
             do
             {
                 rc = libssh2_userauth_publickey_fromfile(
-                        driver->session, strZ(param.user), strZNull(param.keyPub) == NULL ? NULL : strZ(param.keyPub), strZ(param.keyPriv),
-                        strZNull(param.keyPassphrase) == NULL ? NULL : strZ(param.keyPassphrase));
+                        driver->session, strZ(param.user), strZNull(param.keyPub) == NULL ? NULL : strZ(param.keyPub),
+                        strZ(param.keyPriv), strZNull(param.keyPassphrase) == NULL ? NULL : strZ(param.keyPassphrase));
```

Comment:
unrelated change?  Probably should have been an independent commit.

Reply:
OK. I'll have to remember narrow my commits to related changes.